package.json

```diff
@@ -52,20 +52,40 @@
     "migrate": "migrate up",
     "migrate-down": "migrate down",
     "update": "npm install --quiet && npm prune && migrate",
-    "postinstall": "bower install --allow-root && bower prune --allow-root",
     "dashboard:mail": "maildev",
     "dashboard:jobs": "agendash --db=mongodb://localhost/trustroots-dev --collection=agendaJobs --port=1081",
     "agenda-maintenance": "node ./bin/db-maintenance/archive-done-agenda-jobs.js"
   },
   "dependencies": {
     "acl": "~0.4.11",
     "agenda": "~1.0.3",
+    "angular-animate": "1.5.11",
+    "angular-aria": "1.5.11",
+    "angular-chosen-localytics": "~1.9.0",
+    "angular-confirm": "~1.2.6",
+    "angular-loading-bar": "~0.9.0",
+    "angular-locker": "~2.0.4",
+    "angular-mailcheck": "~0.2.0",
+    "angular-message-format": "1.5.11",
+    "angular-mocks": "1.5.11",
+    "angular-moment": "~1.3.0",
+    "angular-resource": "1.5.11",
+    "angular-sanitize": "1.5.11",
+    "angular-touch": "1.5.11",
+    "angular-trustpass": "~0.4.0",
+    "angular-ui-bootstrap": "~2.5.0",
+    "angular-ui-router": "~0.3.2",
+    "angular-waypoints": "~2.0.0",
+    "angular": "1.5.11",
+    "angulargrid": "~0.6.0",
+    "angulartics-google-analytics": "~0.5.0",
+    "angulartics": "~1.6.0",
     "async": "~2.6.1",
     "autolinker": "~1.7.1",
     "autoprefixer": "~9.1.5",
     "basic-auth": "~2.0.1",
     "body-parser": "~1.18.3",
-    "bower": "~1.8.4",
```

Comment:
Bower is still used inside `message-center` lib in the postinstall script [1], and install fails if you don't have it (I guess you have bower globally installed?)

[1] https://github.com/e0ipso/message-center#

Reply:
Good catch!

We should probably just include `message-center` directly in Trustroots codebase; it's unmaintained anyway.